lab03-tom/lab03-tom/Program.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System;
+using System.IO;
+
+namespace lab03_tom
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+
+            Console.WriteLine("Welcome to Josie's Word Guess Game!");
+            InputHandler();
+            Console.Read();
+        }
+
+        static public int HomeNav()
+        {
+            try
+            {
+                Console.WriteLine("What would you like to do? Enter a number:");
+                Console.WriteLine("1. Play");
+                Console.WriteLine("2. View text file");
+                Console.WriteLine("3. Add text");
+                Console.WriteLine("4. Delete text");
+                Console.WriteLine("5. Exit");
+
+                int input = Convert.ToInt32(Console.ReadLine());
+                return input;
+            }
+            catch(FormatException)
+            {
+                Console.WriteLine("Enter a number");
+            }
+            return 0;
+        }
+
+        static public void InputHandler()
+        {
+            string filePath = @"C:\Users\Tom\source\repos\Day-3-WordGuessGame\MysteryWords.txt";
```

Comment:
You should use a relative path to your source directory such as `..\MysteryWords.txt` or `$"{Environment.CurrentDirectory}\\..\\MysteryWords.txt"`